src/main/java/com/impactupgrade/nucleus/service/segment/XeroAccountingPlatformService.java

```diff
@@ -493,12 +442,37 @@ protected List<LineItem> getLineItems(AccountingTransaction accountingTransactio
         if (accountingTransaction.transactionType == EnvironmentConfig.TransactionType.TICKET) {
             lineItem.setAccountCode("160");
             lineItem.setItemCode("EI");
```

Comment:
@VSydor Picky: put these in an `else` below the following `if`, instead of setting them twice?